draft-kazuho-httpbis-priority.md

```diff
@@ -731,6 +733,10 @@ Mike Bishop, Roberto Peon, Robin Marx, Roy Fielding.
 
 # Change Log
 
+## Since draft-kazuho-httpbis-priority-03
+
+* Changed numbering from [-1,6] to [0,7] (#78)
```

Comment:
```suggestion
* Urgency parameter value range shifted to zero-base index
   (from [-1,6] to [0,7]) (#78)
```

Reply:
I'll apply this manually in an editorial cleanup.

Reply:
(y) thanks